targets/f7/api_symbols.csv

```diff
@@ -1,5 +1,5 @@
 entry,status,name,type,params
-Version,+,61.1,,
+Version,+,61.2,,
```

Comment:
We need to keep our API version in sync with OFW. When we add or change something, API version should stay the same. Side note, icons being in API is exclusive to momentum and xfw due to how asset packs work

Reply:
Gotcha gotcha, makes sense. Fine to manually decrement?

Reply:
Yep!